src/assets/toolkit/scripts/lib/component/fact-slider.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+export class FactSlider {
+  constructor (element, {
+    facts = element.querySelectorAll('.FunFact'),
+    factNavNext = element.querySelector('.js-FunFact-next'),
+    factNavPrev = element.querySelector('.js-FunFact-prev'),
+    factNavCurrent = element.querySelector('.js-FunFact-current'),
+    factNavTotal = element.querySelector('.js-FunFact-total'),
+    className = 'is-visible'
+  } = {}) {
+
+    facts = Array.from(facts);
+    let numFacts = facts.length;
+    let counter = 0;
+    factNavTotal.innerHTML = numFacts;
+
+    Object.assign(this, {
+      element,
+      facts,
+      numFacts,
+      factNavNext,
+      factNavPrev,
+      factNavCurrent,
+      factNavTotal,
+      className,
+      counter
+    });
+
+    this.navigate();
+  }
+
+  showCurrent () {
+    // Get the index of the current item
+    let factToShow = Math.abs(this.counter % this.numFacts);
```

Comment:
Unless you want to allow `factToShow` to _change_ at any time in this function, use `const` instead of `let`.